sms_marketing/overview/sms_mailing_reports.rst

```diff
@@ -58,42 +49,41 @@ Note that you are allowed to select more than one *Mailing list*.
 .. note::
    Double-check that you have the phone number of your contacts saved.
 
-
 Sending my mailing
 ==================
 
 .. image:: ./media/sms_marketing6.png
 
-Once you have your mailing created, you must choose when you would like it to be delivered:
-
-- Put in queue: the mailing will be triggered with the next automatic run. Interesting option
-  if you do not mind when the triggering will happen.
-- Send now: sends the message immediately. Advised to use this option if there are not that
-  many recipients.
-- Schedule: allows you to choose a day and time. Best option for mailings that you would like to
-  send during a specific event, to promote an offer while it is active or to simply plan your
-  content strategy in advance.
-- Test: allows you to send an SMS to one or multiple numbers for test purposes. Check, for
-  example, if the link provided lands on the right page while making sure they are responsive. If
-  sending it to multiple numbers, you must remember to use a comma between them.
+| Once you have your mailing created, you must choose when you would like it to be delivered:
 
+| - *Put in Queue*: the mailing is triggered with the next automatic run. Interesting option
+   if you do not need the message to be sent at a specific time.
+| - *Send Now*: sends the message immediately. Use this option if the recipient list is not
+   excessive.
+| - *Schedule*: allows you to choose a day and time. Best option for mailings that you would like to
+   send during a specific event, to promote an offer while it is active, or to simply plan your
+   content strategy in advance.
+| - *Test*: allows you to send an SMS to one or multiple numbers for test purposes. Check, for
+   example, if the link provided lands on the right page while making sure they are responsive. If
+   sending it to multiple numbers, you must remember to use a comma between them.
```

Comment:
I don't understand this part. Why would you want to hard-code your own `-` as bullet points instead of using the default ones ?